src/Week1/HW01.hs

```diff
@@ -0,0 +1,53 @@
+{-# OPTIONS_GHC -Wall #-}
```

Comment:
You can remove this line from here.
We have the -Wall enabled by default in cabal file (see `ghc-options: -Wall ...` in cabal file).